dotcms-integration/src/test/java/com/dotcms/rest/api/v1/workflow/WorkflowResourceIntegrationTest.java

```diff
@@ -2196,18 +2196,14 @@ public void Test_Create_Instance_Of_Content_With_Numeric_Fields_Verify_Message_W
 
             final FireActionForm fireActionForm2 = new FireActionForm(builder2);
             final HttpServletRequest request2 = getHttpRequest();
+
             final Response response2 = workflowResource
                     .fireActionSinglePart(request2, new EmptyHttpResponse(), SAVE_ACTION_ID,
                             brandNewContentlet.getInode(), null, "FORCE", "-1", fireActionForm2);
 
-            final int statusCode2 = response2.getStatus();
-            assertEquals(Status.BAD_REQUEST.getStatusCode(), statusCode2);
-            final ResponseEntityView errorEntityView = ResponseEntityView.class.cast(
-                    response2.getEntity());
-            assertEquals(1, errorEntityView.getErrors().stream()
-                    .filter(errorEntity -> "required".equals(
-                            ErrorEntity.class.cast(errorEntity).getErrorCode())).count());
-
+            fail("Should have thrown an exception");
```

Comment:
When we expect that it throws an exception we do like:
```
@Test(expected = IllegalArgumentException.class)
```

Reply:
I know, but it is exact match exception, the exception comes from a deeper layer and it is being enclosed on many others and the test do not work, it is the reason I need to do this explicit